### DISCUSSION TITLE
How to highlight a div with a border.

### DISCUSSION THREAD

**Discussion by RaSabirov** [requarks/wiki#6834] [Help / Questions]
I want to highlight this field with a border; for example, I highlighted it in red. Could you please advise on the code to use and where to insert it?
![Безымянный](https://github.com/requarks/wiki/assets/89607273/7020fd3e-cf8a-4810-b59c-b0d5109d1414)

**Comment by AlexJFox** [comment]
You can achieve this by using the CSS Override in Code Injection in the Admin > Theme section.

```
#app nav a[href="/en/home"]
{
border: 2px solid red;
}
```

Change the `/en/home` to the path the page links to.
